store/activities.js

```diff
@@ -2,31 +2,45 @@ const config = [
   {
     id: 'overview',
     path: 'overview',
+    icon: 'dashboard',
```

Comment:
I do not agree. Store should know nothing about how it looks.

Reply:
Well, I was thinking about moving this config file somewhere else. For example to a new directory called `configs`?